### PR TITLE
Feature/jmeter report

### LINKED ISSUE #12: Improve the current presentation of results.
It would be useful to add new fields and improve existing ones for better analysis, for example of the throughput per second.

### LINKED ISSUE #14: Adding the option to consume datasets.
It would be useful to variabilize the requests with existing data in the database, this in order to simulate real requests.

### LINKED ISSUE #29: Generate jmeter logs for metrics
We need logs in jmeter style format as input for metrics

### PR DESCRIPTION
**Pull request by juancgalvis**
Features:
- Export request results in jmeter csv format

Fix #29 
Fix #14 
Fix #12

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.13.0-rc.1-alpine
+FROM 1.13.3-alpine
```
Comment (juancgalvis) [inline Dockerfile:1]: Should you add elixir:1.13.3-alpine @JuanDGiraldoM ??

Comment (JuanDGiraldoM) [reply Dockerfile]: Fixed!

**Review by JuanDGiraldoM — APPROVED** [review]

**Pull request merged**